ci-operator/jobs/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,63 @@
 presubmits:
   openshift/cluster-openshift-apiserver-operator:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws
+    decorate: true
+    name: pull-ci-openshift-cluster-openshift-apiserver-operator-master-e2e-aws
+    rerun_command: /test e2e-aws
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
This is specific to the installer repo and our `docs/` (#1772), which the API-server operator [doesn't have][1].  They might have other sections of their repo that they want to ignore for e2e-tests, but for now we should probably just drop this line.

[1]: https://github.com/openshift/cluster-openshift-apiserver-operator/tree/c7a59d4862e76280cb47a52252a4d620ca67c621